src/main/java/hudson/plugins/analysis/core/BuildHistory.java

```diff
@@ -146,13 +152,9 @@ private boolean isSuccessfulAction(final ResultAction<? extends BuildResult> act
     /**
      * Returns the result action of the specified build that should be used to
      * compute the history.
-     *
-     * @param build
-     *            the build
-     * @return the result action
```

Comment:
I deleted them because I think they were redundant, all the information provided by that javadocs is already present in the method prototype. Moreover, the original javadocs are being kept in the deprecated method (below).